src/client/service.js

```diff
@@ -0,0 +1,23 @@
+
+angular
+	.module('tl')
+	.service('tl.client.service', ['tl.service', 'tl.client.resource', function(Service, Client){
+
+		var ClientService = function(){};
+
+		/**
+		 * Generate new merchant payment token (for use with Braintree API)
+		 */
+		ClientService.prototype.paymentToken = function(success, error) {
+			return Client.paymentToken({}, success, error);
```

Comment:
I vote we return it all the time and get rid of the success and error callbacks as last two arguments for all old functions. I'd also like to update the old to take an options arguments as I find I often need to pass more things and they currently don't support it